readme.md

```diff
@@ -296,6 +296,28 @@ metalsmith(__dirname).use(
 
 </details>
 
+#### options.htmlExtensions
+
+Type: `string[]`
+Default: `[".html"]`
+
+A list of all file extensions that should be considered HTML files. When files
+do not end in one of these extensions, it will not be allowed to be used as a
+redirection source.
+
+<details><summary>Example</summary>
```

Comment:
You are missing the closing `</details>`.

Reply:
Added.